learning/tour-of-beam/frontend/lib/components/footer.dart

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../config/theme/colors_provider.dart';
+import '../constants/font_weights.dart';
+import '../constants/links.dart';
+import '../constants/sizes.dart';
+
+class Footer extends StatelessWidget {
+  const Footer({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: ThemeColors.of(context).secondaryBackground,
+      width: double.infinity,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          vertical: kSmSpacing,
+          horizontal: kXlSpacing,
+        ),
+        child: Wrap(
+          spacing: kXlSpacing,
+          crossAxisAlignment: WrapCrossAlignment.center,
+          children: [
+            TextButton(
+              style: TextButton.styleFrom(
+                textStyle: const TextStyle(fontWeight: kNormalWeight),
+              ),
+              onPressed: () {
+                launchUrl(Uri.parse(kReportIssueLink));
+              },
+              child: Text('footer.reportIssue'.tr()),
+            ),
+            TextButton(
+              style: TextButton.styleFrom(
+                textStyle: const TextStyle(fontWeight: kNormalWeight),
```

Comment:
If the style is repeated, can we make it reusable? Some ideas are:
- Make a global final.
- Extract the whole button to a widget. It could be `TranslatedTextButton` or something like that.
- Hang it in a theme so `TextButton` picks it up by default.